### ISSUE TITLE
Fix false positives for sourceMappingURLs

### ISSUE THREAD

**Issue by kanlukasz** [stylelint/stylelint#4667]
> Clearly describe the documentation issue

I can't find any information about preventing "sourceMappingURL" removal
I have a file that contains ```/*# sourceMappingURL=bootstrap.css.map */``` in the last line of file.
Stylelint always deletes it. I would like to prevent this

> What solution would you like to see?

Maybe any specific rule for this?

> My current `stylelintrc` file:

```
{
  "rules": {
	"indentation": 2
  }
}
```
> I tried with ```"extends": "stylelint-config-twbs-bootstrap/css"``` 

but still the same

**Comment by jeddy3** [comment]
@kanlukasz Thanks for the report.

I can reproduce this locally.

This issue hasn't come up before. I suspect because stylelint is usually run on source files (which don't contain source map URLs). What is your use case for checking compiled CSS?

Regardless of your use case, stylelint shouldn't remove this line when fixing. It is also problematic for linting where stylelint treats it as an empty line (see [this demo](https://stylelint.io/demo#N4Igxg9gJgpiBcID0AqAxAAgM4QK4CcwYBZAQwAdyBLAOwHMBVAJQBkBeAIwggBcsf8FAHRgsWIQFsKGFEgA6NEABoQWAJ40epAB4JwY5eAg0AZlTp7gCjBjkh8uADYwsd+Bis0bNu1O0BaGAlyHjV-R1oXNwwARmsMAF8FBJAEoA)). I'm not sure why this is, as the [AST has it as a comment](https://astexplorer.net/#/gist/7a535eac5aebe0e93e2b43e1d4e1b0a8/51ae932e52ce6b424ba79ef6da0d417119b8501f).

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

**Comment by jeddy3** [comment]
> This issue hasn't come up before. 

My bad. It has come up before. Closing a duplicate of https://github.com/stylelint/stylelint/issues/3802. That issue is label as "ready to implement" and is looking for someone to champion it.
